addons/edi/models/edi_document.py

```diff
@@ -127,6 +127,9 @@ def _default_project_id(self):
         string="Sequential Execution Required", help="Document can only be processed if all "\
         "previous documents of the same type have been completed", default=False
     )
+    one_file_per_attachment = fields.Boolean(
```

Comment:
From the help i seems like the field would be better to be called one_edi_per_attachment or one_document_per_attachment